pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1356,7 +1356,7 @@
           ^-  move
           :^  duct  %pass  /channel/poke/[channel-id]/(scot %ud request-id.i.requests)
           =,  i.requests
-          [%g %deal `sock`[our ship] `internal-task:gall`[app %punk mark %json !>(json)]]
```

Comment:
This change will break our other web apps that have not been similarly ported to this paradigm. What you should probably do instead is implement an alternative poke api in channel.js and make both options available.